datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -216,6 +216,55 @@ def __init__(self, *args, **kwargs):
         logger = logging.getLogger('{}.{}'.format(__name__, self.name))
         self.log = CheckLoggingAdapter(logger, self)
 
+        exclude_metrics_filters = self.instance.get('exclude_metrics_filters', []) if instance else []
```

Comment:
This new logic is used when constructing and validating `exclude_labels` and `include_labels`: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/scraper.py#L100, as well as `exclude_metrics`: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/scraper.py#L132